src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -376,9 +437,33 @@ protected override void BeginProcessing()
 
             if (!_fragment)
             {
-                WriteObject("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\"  \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">");
+                if(!_transitional)
+                {
+                    WriteObject("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\"  \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">");
+                }
+                else
+                {
+                    WriteObject("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\"  \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">");
+                }
                 WriteObject("<html xmlns=\"http://www.w3.org/1999/xhtml\">");
+                if(_metaSpecified)
+                {
+                    foreach(string s in _meta.Keys)
+                    {
+                        if(s != "content-type" && s != "default-style" && s !="refresh")
+                        {
+                            WriteObject("<meta name=\"" + s + "\" content=\"" + _meta[s] + "\">");
+                        }
+                        else {
+                            WriteObject("<meta http-equiv=\"" + s + "\" content=\"" + _meta[s] + "\">");
+                        }
+                    }
+                }
                 WriteObject("<head>");
+                if(_charsetSpecified)
+                {
+                    WriteObject("<meta charset=\"" + _charset + "\">");
```

Comment:
Validation would be great. I don't see too many changes in the charset list but it would be great if the validation were dynamic pulling from w3schools or something. I was trying to find documentation on what the browser does when a non standard charset is specified; such as "foo". I believe it analyzes the html and determines the best or just defaults to UTF-8 for HTML5.